Userland/Libraries/LibSoftGPU/Clipper.cpp

```diff
@@ -91,6 +91,20 @@ FLATTEN static constexpr void clip_plane(Vector<GPU::Vertex>& input_list, Vector
     }
 }
 
+void Clipper::clip_points_against_frustum(Vector<GPU::Vertex>& vertices)
+{
+    Vector<GPU::Vertex> output_vertices;
+    for (auto& vertex : vertices) {
+        auto const coords = vertex.clip_coordinates;
+        if (point_within_clip_plane<ClipPlane::LEFT>(coords) && point_within_clip_plane<ClipPlane::RIGHT>(coords)
+            && point_within_clip_plane<ClipPlane::TOP>(coords) && point_within_clip_plane<ClipPlane::BOTTOM>(coords)
+            && point_within_clip_plane<ClipPlane::NEAR>(coords) && point_within_clip_plane<ClipPlane::FAR>(coords))
+            output_vertices.append(vertex);
+    }
+
+    vertices = output_vertices;
```

Comment:
`move()` this to avoid an extra reallocation here...

Reply:
Thanks - instead of `move()` I went for `clear_with_capacity` + `extend`. See comment on other remark!